### ISSUE TITLE
Launching meteor does not show any error logs.

### ISSUE THREAD

**Issue by mudum** [PEM--/meteor-helper#45]
This may be windows' fault. I am on windows 10 64-bit. When I launch my app using atom, it doesn't show anything except for the warning icon continuing to blink. Although it can run when just using the command prompt.

**Comment by PEM--** [comment]
It's also confirmed on OSX. Something has changed in the last Atom's update. Weird.

Thanks for the warning.

**Comment by sdinesh86** [comment]
open and close developer tools once you launch meteor, the logs will start to show up.

**Comment by mudum** [comment]
It now gives me the tasklist error

**Comment by cecet** [comment]
Please see if mongo process was killed properly, solved the same problem for me – it was due to wrongly seth paths and application crashing before killing it. 

About those paths – I found that if I already have meteor directory added to system `PATH` I coud just  set "Meteor Path" in extension settings to:

```
meteor.bat
```

and it just worked.

(Please note that you do not need to create or modify any file on your filesystem in this solution)

And if this doesn't work then probably there is no meteor directory in system path variable – you can fix it by adding it permanently by typing in command line:

```
SET PATH=%PATH%;C:\Users\username\AppData\Local\.meteor\
```

(You can remove it later if you want of course, but it's really good thing to have; and if you want to use several different versions of meteor, I would suggest just creating many .bat files pointing to them in the meteor directory that was added to path, and just edit their contents to create/overwrite default meteor.bat file to point to different installs inside, and later just delete default meteor.bat, to now use unknown installation in case of something crashing... but these are just my loose thoughts).

**Comment by zgudino** [comment]
@sdinesh86 Thanks! That solve the problem

**Comment by PEM--** [comment]
Fixed in 0.27.0.

**Comment by mudum** [comment]
excellent
